test/bigfloat_fma.jl

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2023 MutableArithmetics.jl contributors
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v.2.0. If a copy of the MPL was not distributed with this file, You can obtain
+# one at http://mozilla.org/MPL/2.0/.
+
+@testset "fma $op output values" for op in (MA.operate_to!, MA.operate_to!!),
+    i in 1:100
```

Comment:
I prefer a few fixed tests for which we know `muladd` is different than `fma` than 100 random tests since it's difficult to reproduce

Reply:
Now the testing approach is to try a range of two-bit `BigFloat` values.